environments/test.env

```diff
@@ -1,6 +1,6 @@
 ENV="test"
 JWT_SECRET="secret"
-JWT_VALIDITY_IN_DAYS=7
+JWT_VALIDITY_IN_HOURS=24
```

Comment:
why from 7 days to 24 hours

Reply:
This is for test and we are not using JWT_VALIDITY_IN_DAYS in our code you can have a look on dev.env